packages/calcite-components/src/components/input/common/_mixins.scss

```diff
@@ -0,0 +1,532 @@
+/**
+ * CSS Custom Properties
```

Comment:
How would this mixin be documented in the doc site?

Reply:
This is here for reference. As you can see, the JSDocs are UTD for each component.